{"name": "test_sign_up_invalid_phone_num2", "status": "passed", "description": "the test is verify, that user can't sign up with invalid phone number", "start": 1680369626718, "stop": 1680369627429, "uuid": "b335a348-59f6-4c5c-b749-cbdff022dd54", "historyId": "3b1e7642cb70f5d9b866e5fb6b27472f", "testCaseId": "83d801c547d346e613bdec82c5721eea", "fullName": "tests.chromeTests.testSignUpModal.TestSignUpModal#test_sign_up_invalid_phone_num2", "labels": [{"name": "parentSuite", "value": "tests.chromeTests"}, {"name": "suite", "value": "testSignUpModal"}, {"name": "subSuite", "value": "TestSignUpModal"}, {"name": "host", "value": "krot"}, {"name": "thread", "value": "8896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.chromeTests.testSignUpModal"}]}